errtracker/errtracker.go

```diff
@@ -64,10 +70,19 @@ func Report(snap, errMsg, dupSig string, extra map[string]string) (string, error
 		return "", nil
 	}
 
-	machineID, err := ioutil.ReadFile(machineID)
-	if err != nil {
-		return "", err
+	var machineID []byte
+	var err error
+	for _, id := range machineIDs {
+		machineID, err = ioutil.ReadFile(id)
+		if err == nil {
```

Comment:
I think we should only silently ignore the error when the file does not exist and log all other errors.